azure_li_services/defaults.py

```diff
@@ -51,3 +53,22 @@ def get_config_file(self):
             raise AzureHostedConfigFileNotFoundException(
                 'No Azure Li/VLi file found: {0}'.format(config_file)
             )
+
+    @classmethod
+    def mount_config_source(self):
+        config_type = namedtuple(
+            'config_type', ['name', 'location', 'label']
+        )
+        azure_config = config_type(
+            name=os.path.basename(Defaults.get_config_file_name()),
+            location='/mnt', label='azconfig'
+        )
+        try:
+            Command.run(
+                ['mount', '--label', azure_config.label, azure_config.location]
+            )
+        except Exception:
+            Command.run(
+                ['mount', '/dev/dvd', azure_config.location]
+            )
```

Comment:
If this fails, i.e. no DVD is present either, then an exception is triggered that, as far as I can see is not caught and thus a traceback would bubble up to the top level. I think we should exit more gracefully.

Reply:
Command.run will throw an AzureHostedCommandException with the error information from the command. That's imho good enough, or do you want to express that we tried first with the lun and that the fallback also didn't work ?